kedro/extras/extensions/ipython.py

```diff
@@ -77,35 +89,21 @@ def reload_kedro(path, env: str = None, extra_params: Dict[str, Any] = None):
         logging.info("Registered line magic `%s`", line_magic.__name__)  # type: ignore
 
 
-def init_kedro(path=""):
-    """Line magic to set path to Kedro project.
-    `%reload_kedro` will default to this location.
-    """
-    global project_path
-    if path:
-        project_path = Path(path).expanduser().resolve()
-        logging.info("Updated path to Kedro project: %s", str(project_path))
-    else:
-        logging.info("No path argument was provided. Using: %s", str(project_path))
-
-
 def load_ipython_extension(ipython):
     """Main entry point when %load_ext is executed"""
 
-    global project_path
-    global startup_path  # pylint:disable=global-variable-not-assigned
+    global default_project_path
 
-    ipython.register_magic_function(init_kedro, "line")
     ipython.register_magic_function(reload_kedro, "line", "reload_kedro")
 
-    project_path = _find_kedro_project(startup_path)
+    default_project_path = _find_kedro_project(Path.cwd())
```

Comment:
**See following comment no error handling before this. If you agree with what I say there then this one is redundant...** 

Minor: If `_find_kedro_project` can't find a kedro project and returns `None` then this sets the `default_project_path` to `None` and will give the message "No path argument was provided. Using: None".

To me it would just feel a bit more natural if we used the `None` value to specifically mean "`reload_kedro` should re-use the current `default_project_path`" and instead just passed `Path.cwd()` in here instead. It's the default value of `default_project_path` set on L14 so seems a bit odd to change it to `None` rather than just keeping that default.

```suggestion
    kedro_project path = _find_kedro_project(Path.cwd())
    default_project_path = kedro_project_path or Path.cwd()
```

Reply:
Yeah that's actually a fair point, I'll leave it to use cwd over None.